youtube-8m/README.md

```diff
@@ -1,655 +1,7 @@
-# YouTube-8M Tensorflow Starter Code
+# YouTube-8M Custom Code
```

Comment:
Can you remove the remnants from our old README? the old read me has been moved to `CUSTOM_CHANGES.md`